### PR TITLE
removing kit log action and menu along with didact and readme references

### PR DESCRIPTION
**Pull request by bfitzpat**
Signed-off-by: bfitzpat@redhat.com <bfitzpat@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -235,7 +235,7 @@ Optionally, to disable status bar messages:
 
 ## Viewing the log for a Apache Camel K Operator
 
-In the background of each Apache Camel K system, there is usually an Operator coordinating the various builders and integrations being spun up. This log will often contain information not found in the other logs (see [Viewing the running log](#viewing-the-running-log-for-a-published-apache-camel-k-integration) and [Viewing the Kit Builder Log](#viewing-the-kit-builder-log-for-an-apache-camel-k-integration) below for details on opening the other available logs).
+In the background of each Apache Camel K system, there is usually an Operator coordinating the various builders and integrations being spun up. This log will often contain information not found in the operator log (see [Viewing the running log](#viewing-the-running-log-for-a-published-apache-camel-k-integration) below for details on opening the integration log).
```
Comment (apupier) [inline README.md:238]: `not found in the operator log` --> isn't `not found in the integration log`?

**Comment by djelinek** [comment]
Hey @bfitzpat , so understand I it correctly, the issue [FUSETOOLS2-296](https://issues.redhat.com/browse/FUSETOOLS2-296) is no more needed after these changes right?

**Comment by bfitzpat** [comment]
> Hey @bfitzpat , so understand I it correctly, the issue [FUSETOOLS2-296](https://issues.redhat.com/browse/FUSETOOLS2-296) is no more needed after these changes right?

Yep! We may add the functionality back in at some point, but for now this is off the table. Thanks for verifying!!

**Comment by djelinek** [comment]
OK, I will link this issues together in JIRA to easily track for the future and for now I am closing also UI test task. Thank for clarifying.

**Pull request merged**